integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -84,6 +86,69 @@ public void testAddDate() throws IOException {
     verifyDataRows(result, rows("2020-09-17"));
   }
 
+  @Test
+  public void testConvert_TZ() throws IOException {
```

Comment:
What is the importance of each test case? You could break them out into separate tests (🔥) or at least document each one.